app/templates/admin/createEvent.html

```diff
@@ -20,20 +20,33 @@
 {% set isNewEvent = 'create' in request.path %}
 <!-- Program manager isCeltsAdmin and they are a manager for the program  -->
 <div class="text-center mb-3">
-    {% if isNewEvent %}
-      {% if template.tag == 'single-program' %}
-        {% set page_title = 'Create Event for ' + eventData["program"].programName %}
-      {% elif template.tag == 'all-volunteer' %}
-        {% set page_title = 'Create All Volunteer Training' %}
-      {% elif template.tag == 'no-program' %}
-        {% set page_title = 'Create Other CELTS-Sponsored Event' %}
-      {% else %}
-        {% set page_title = 'Create ' + template.name + ' Event' %}
-      {% endif %}
+  {% if isNewEvent %}
+    {% if template.tag == 'single-program' %}
+      {% set page_title = 'Create Event for ' + eventData["program"].programName %}
+    {% elif template.tag == 'all-volunteer' %}
+      {% set page_title = 'Create All Volunteer Training' %}
+    {% elif template.tag == 'no-program' %}
+      {% set page_title = 'Create Other CELTS-Sponsored Event' %}
+    {% else %}
+      {% set page_title = 'Create ' + template.name + ' Event' %}
+    {% endif %}
+    {% else %}
+      {% set chicken = eventData["program"].programName %}
+      {% if eventData.name == 'All Volunteer Training' and program == None %}
+        <h1>{{eventData.name}}</h1>
+        <h3> An All Volunteer Training Event</h3>
+      {% elif eventData.program != None %}
+        <h1>{{eventData.name}}</h1>
+        <h3>A {{chicken}} Event</h3>
```

Comment:
Can these "chicken" variables have a more descriptive name? Something like "programName" or similar should good, as long as it doesn't clash with other variables in the file.

Reply:
my bad!! completely forgot to change it